metadata-models/src/main/pegasus/com/linkedin/ml/metadata/IntendedUse.pdl

```diff
@@ -0,0 +1,22 @@
+namespace com.linkedin.ml.metadata
+
+/**
+ * Intended Use for the ML Model
+ */
+record IntendedUse {
+
+  /**
+   * Primary Use cases for the model.
+   */
+  primaryUses: optional string
```

Comment:
can we define this as an enum if the possible values are small?

Reply:
Primary Intended Uses could be defined per the domain the model is developed. The uses can be broad per domain or any specifics within the domain.

Reply:
I think in that case it should at least be an array of string to capture multiple uses?